simpleclient_hibernate/pom.xml

```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>[4.13.1,)</version>
```

Comment:
Please stick with a specific version, and keep it the same in all the poms.

Reply:
Done. It looks like these two were the only ones left behind.

Reply:
Hmm, what about spring_web?